Implemented map name without having full path in map name (#78)
l3af-project/l3afd: docs/api/README.md

| name                | string                                         | ratelimiting                                                   | Name of the eBPF Program                                                                                                         |
| seq_id              | number                                         | `1`                                                            | Position of the eBPF program in the chain. Count starts at 1.                                                                    |
| artifact            | string                                         | `"l3af_ratelimiting.tar.gz"`                                   | Userspace eBPF program binary and kernel eBPF byte code in tar.gz format                                                         |

> | map_name            | string                                         | `"ep1_next_prog_array"`                            | Chaining program map in the file system. This should match the eBPF program code.                                      |

Reviewer: ```suggestion
| map_name            | string                                         | `"ep1_next_prog_array"`                            | Chaining program map to pin to. This should match the eBPF program code.                                      |
```

| artifact            | string                                         | `"l3af_ratelimiting.tar.gz"`                                   | Userspace eBPF program binary and kernel eBPF byte code in tar.gz format                                                         |
| map_name            | string                                         | `"ep1_next_prog_array"`                            | Chaining program map to pin to. This should match the eBPF program code.                                      |
=======
| artifact            | string                                         | `"l3af_ratelimiting.tar.gz"`                                   | Userspace eBPF program binary and kernel eBPF byte code in tar.gz format     |
| ebpf_package_repo_url | string         | `"https://l3af.io/"`     | eBPF package repository URL.  If it is not provided default URL is used.|                                                  |
| map_name            | string                                         | `"/sys/fs/bpf/ep1_next_prog_array"`                            | Chaining program map in the file system with path. This should match the eBPF program code.                                      |

> >>>>>>> main

Author: removed
